### PR TITLE
React task 1 update

### PR DESCRIPTION
**Pull request by chelovek-gorod**

### CODE REVIEW

**Comment by broshkabro** [comment]
1. Study the git-flow. You should not commit to `the main` branch directly. 
2. The structure of your app should be the following:
image
3. Create a separate folder for the components and move them into it. Move all images into a separate folder as well. You should get the following folder structure:
image

**Review by broshkabro — CHANGES_REQUESTED** [review]
please address the comments above

**Comment by chelovek-gorod** [comment]
Done

File: src/App/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import UserDetails from '../UserDetails/UserDetails';
+import Content from '../Content/Content';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```
Comment (broshkabro) [inline src/App/App.js:7]: please use kebab case for class names

File: src/App/App.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+      <UserDetails />
+
```
Comment (broshkabro) [inline src/App/App.js:10]: remove extra spaces

File: src/components/Avatar/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import avatar from '../../img/avatar.png';
+import './Avatar.css';
+
+function Avatar() {
+  return (
+    <div className="avatar border">
+      <img src={avatar} alt="Avatar"/>
```
Comment (broshkabro) [inline src/components/Avatar/Avatar.js:7]: Avatar must be a part of the data object. You may have plenty of different users. All of them may have their own avatar.

File: src/components/Userinfo/Userinfo.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import './Userinfo.css';
+
+const user = {
```
Comment (broshkabro) [inline src/components/Userinfo/Userinfo.js:3]: You have to pass user data as a prop according to the task.

**Review by broshkabro — APPROVED** [review]

**Pull request merged**